lib/components/app/responsive-webapp.js

```diff
@@ -231,17 +235,24 @@ class RouterWrapperWithAuth0 extends Component {
               ]}
               render={() => <WebappWithRouter {...this.props} />}
             />
+            <Route
+              component={FavoritePlaceScreen}
+              path={`${PLACES_PATH}/:id`}
```

Comment:
When creating a place in the context of a new account, I think we should avoid placing the My account header content at the top of the page. Doing so takes the user out of the create account context and also shows the Trips/Settings links, which should not be available at this stage. I don't want to overcomplicate this though... thoughts on a clean approach here?

![image](https://user-images.githubusercontent.com/2370911/107789476-a4ba0400-6d1f-11eb-9116-c746147e6c05.png)

Reply:
This is now addressed in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/70f449c2c26d870e4d2a4e6c3a7e77bd4fa5b63a. Per chat, the Back/Save buttons are harmonized in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/411caf13976af1e18b2f2b18a255f4f445547838.